src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -57,6 +57,14 @@ public function countByCustomerAndCoupon(CustomerInterface $customer, CouponInte
     public function countByCustomerAndPaymentState(CustomerInterface $customer, $state);
 
     /**
+     * @param string $customerEmail
+     * @param string $state
+     *
+     * @return int
+     */
+    public function countByCustomerEmailAndPaymentState($customerEmail, $state);
```

Comment:
This method can be removed. See https://github.com/Sylius/Sylius/pull/5363/files#r68964331.